environments/icds-cas/postgresql.yml

```diff
@@ -119,6 +118,66 @@ dbs:
       name: warehouse
       host: pgwarehouse0
       django_migrate: True
+    - name: commcarehq_p10
```

Comment:
I think these can be deleted now. They were only here so that we could set them up in a separate step